var/spack/repos/builtin/packages/libevent/package.py

```diff
@@ -54,6 +58,9 @@ def libs(self):
         libs = find_libraries("libevent", root=self.prefix, shared=True, recursive=True)
         return LibraryList(libs)
 
+    def autoreconf(self, spec, prefix):
+        which("bash")("autogen.sh")
+
```

Comment:
```suggestion
    def autoreconf(self, spec, prefix):
        Executable("./autogen.sh")()

```

It looks like libevent/autogen.sh seems to support any posix shell so we should execute it here instead of running with bash to allow for more compatibility with Linux distributions that don't include bash in their base image.